doc/mesh_filter/mesh_filter_tutorial.rst

```diff
@@ -0,0 +1,111 @@
+Mesh Filter with UR5 and Kinect
+===============================
+
+MoveIt's mesh filter functionality removes your robot's geometry from its point cloud! If your robot's arm is in your depth sensor's view, the points associated with the arm are subtracted from the point cloud. 
+
+This is accomplished by giving the original point cloud, the robot's transforms (tf) and the robot's URDF as inputs.
+The filter then published a modified point cloud which does not contain the points that overlaps with the current robot state.
+
+.. image:: MeshFilter.gif
+
+Getting Started 
+---------------
+
+* Follow the instructions for :moveit_website:`installing MoveIt<install>`
+  first if you have not already done that. 
+
+* Clone the `Universal Robot package <https://github.com/ros-industrial/universal_robot>`_ to your workspace for Melodic.
+
+* Install `Gazebo <http://gazebosim.org/tutorials?tut=install_ubuntu&cat=install>`_
```

Comment:
Something to point out in a note is that after doing this I also needed to install one more thing for the kenect plugin:

```
sudo apt install ros-melodic-gazebo-plugins
```